src/openrct2-ui/windows/Guest.cpp

```diff
@@ -486,6 +486,7 @@ namespace OpenRCT2::Ui::Windows
 
         void SetPage(int32_t newPage)
         {
+            // Skip setting page if we're already on this page, unless we're initialising the window
```

Comment:
Shouldn’t this comment be on line +500?

Reply:
Nice catch. Yes, I'm not sure how this happened.